src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/services/service-form/service-form.component.html

```diff
@@ -883,8 +890,121 @@
+              <input id="client_secret"
+                     class="form-control"
+                     type="password"
+                     formControlName="client_secret"
+                     placeholder="proxy">
+              <cd-help-text i18n>Along with Client ID authenticates application to the IDP</cd-help-text>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('client_secret', frm, 'required')"
+                    i18n>This field is required.</span>
+            </div>
+          </div>
+          <!-- oidc_issuer_url -->
+          <div class="form-group row">
+            <label class="cd-col-form-label required"
+                   for="oidc_issuer_url">
+              <span i18n>Issuer url</span>
+            </label>
+            <div class="cd-col-form-input">
+              <input id="oidc_issuer_url"
+                     class="form-control"
+                     type="text"
+                     formControlName="oidc_issuer_url">
+              <cd-help-text i18n>IDP's issuer url</cd-help-text>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('oidc_issuer_url', frm, 'required')"
+                    i18n>This field is required.</span>
+            </div>
+          </div>
+          <!-- https_address -->
+          <div class="form-group row">
+            <label class="cd-col-form-label"
+                   for="https_address">
+              <span i18n>Https address</span>
+            </label>
+            <div class="cd-col-form-input">
+              <input id="https_address"
+                     class="form-control"
+                     type="text"
+                     formControlName="https_address"
+                     placeholder="0.0.0.0:4180">
+              <cd-help-text i18n>Address format [IP|hostname]:port</cd-help-text>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('https_address', frm, 'required')"
+                    i18n>This field is required.</span>
+              <span class="invalid-feedback"
+                    *ngIf="serviceForm.showError('https_address', frm, 'httpsAddressPattern')"
+                    i18n>Format must be [IP|hostname]:port and port between 0 and 65535</span>
+            </div>
+          </div>
+          <!-- redirect_url -->
+          <div class="form-group row">
+            <label class="cd-col-form-label"
+                   for="redirect_url">
+              <span i18n>Redirect URL</span>
+            </label>
+            <div class="cd-col-form-input">
+              <input id="redirect_url"
+                     class="form-control"
+                     type="text"
+                     formControlName="redirect_url"
+                     placeholder="https://<your-ip>:4180/oauth2/callback">
+              <cd-help-text i18n>The URL oauth2-proxy will redirect to after a successful login.</cd-help-text>
```

Comment:
```suggestion
              <cd-help-text i18n>The URL <pre>oauth2-proxy</pre> will redirect to after a successful login.</cd-help-text>
```

Reply:
Any reason for this?

Reply:
`oauth2-proxy` is the name of the service, or add "service" after oauth2-proxy perhaps.